packages/flutter/test/painting/gradient_test.dart

```diff
@@ -332,9 +330,7 @@ void main() {
       ],
     );
 
-    final RadialGradient actual = RadialGradient.lerp(testGradient1, testGradient2, 0.5);
-
-    expect(actual.focal, isNull);
+    final RadialGradient? actual = RadialGradient.lerp(testGradient1, testGradient2, 0.5);
```

Comment:
nit: why not use `!` and leave the null check as is?

Reply:
either works :)